src/botPage/view/react-components/NotoficationBanner.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { translate } from '../../../common/i18n';
+import { isEuCountry, isUKCountry } from '../../../common/footer-checks';
+import { getTokenList, get as getStorage } from '../../../common/utils/storageManager';
+import { AppConstants } from '../../../common/appId';
+
+const NotificationBanner = ({ api }) => {
+    const [showBanner, setShowBanner] = React.useState(false);
+    const tokenList = getTokenList();
+    const landingCompanyName = [];
+
+    React.useEffect(() => {
+        checkForShowBanner(landingCompanyName);
+    }, []);
+
+    const getActiveToken = activeToken => {
+        const activeTokenObject = tokenList.filter(tokenObject => tokenObject.token === activeToken);
+        return activeTokenObject.length ? activeTokenObject[0] : tokenList[0];
+    };
+
+    const checkForShowBanner = () => {
+        if (tokenList.length) {
+            tokenList.map(token => landingCompanyName.push(token.loginInfo.landing_company_name));
+
+            const activeToken = getActiveToken(tokenList, getStorage(AppConstants.STORAGE_ACTIVE_TOKEN));
+
+            if (landingCompanyName.length === 1 && landingCompanyName.includes('virtual')) {
+                isEuUK(false);
+            }
+            if (
+                landingCompanyName.length === 2 &&
+                landingCompanyName.includes('maltainvest') &&
+                landingCompanyName.includes('virtual')
+            ) {
+                setShowBanner(true);
+            }
+            if (
+                landingCompanyName.includes('maltainvest') &&
+                landingCompanyName.includes('virtual') &&
+                (landingCompanyName.includes('malta') || landingCompanyName.includes('iom'))
+            ) {
+                if (activeToken.loginInfo.landing_company_name === 'maltainvest') {
+                    setShowBanner(true);
+                }
+            }
+        } else {
+            isEuUK(api, true);
+        }
+    };
+
```

Comment:
The final code for `checkForShowBanner` will be this 
```suggestion
const checkForShowBanner = () => {
    if (!tokenList.length) {
        isEuUK(api, true);
        return;
    }
    const landingCompanyName = tokenList.map(token => token.loginInfo.landing_company_name);

    const activeToken = getActiveToken(tokenList, getStorage(AppConstants.STORAGE_ACTIVE_TOKEN));

    if (landingCompanyName.length === 1 && landingCompanyName.includes('virtual')) {
        isEuUK(false);
        return
    }
    if (
        landingCompanyName.includes('maltainvest') &&
        landingCompanyName.includes('virtual') && 
        (landingCompanyName.length === 2 || 
            ((landingCompanyName.includes('malta') || landingCompanyName.includes('iom')) &&
        activeToken.loginInfo.landing_company_name === 'maltainvest'))
    ) {
            setShowBanner(true);
            return
    }
 
};
```